### PR TITLE
Hapi v17 Compatibility

### PR DESCRIPTION
**Pull request by jamesdixon**
This PR updates the plugin to be compatible with Hapi v17+. It also updates to the latest version of Rollbar as well as updating other dependencies.

### CODE REVIEW

**Comment by jmonster** [comment]
dang 👏 

I'll try to look at this as soon as I can

**Comment by jamesdixon** [comment]
thanks @jmonster!

**Comment by jamesdixon** [comment]
bump 👍

File: test/log.js [hunk 1]
```diff
@@ -7,44 +7,42 @@ const lab = exports.lab = Lab.script();
 
 let server;
 
+const ROLLBAR_TOKEN = '58b67946b9af48e8ad07595afe9d63b2';
```
Comment (jmonster) [inline test/log.js:10]: Is this a real value you want committed?

Comment (jmonster) [reply test/log.js:10]: ha I see where it came from 🙈

**Comment by jmonster** [comment]
This looks good to me! I'm not working with hapi/nodejs much at the moment so I'm afraid I'll be lazy about this project..so I've added you as a collaborator :)

**Comment by jamesdixon** [comment]
Awesome! Thx for the review and adding me as a collaborator.

Do you want to handle NPM releases or do you want to add me to NPM as well?

**Comment by jmonster** [comment]
published (v5) + added

**Comment by jamesdixon** [comment]
thx. cheers!

**Pull request merged**